guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -0,0 +1,46 @@
+[id="migrating-to-a-new-server_{context}"]
+
+.Prerequisites
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Creating a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 7
+
+Before you perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 system, back up your {ProjectServer} or {SmartProxyServer} data on the {RHEL} 7 system by creating an offline backup.
+
+If you recently created an offline backup, you can perform an incremental backup to update the existing backup.
+
+.Procedure
+* Perform an offline or incremental backup on the source machine:
+** To perform an offline backup, see {AdministeringDocURL}: Administering_Project/{Backing_Up_Server_and_Proxy}.
+** To perform an incremental backup, see {AdministeringDocURL}: Administering_Project/{Performing-an-Incremental-Backup}.
+* Transfer the backup to the target machine.
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} to migrate to {RHEL} 8
+
+The fresh installation method has three steps:
+
+. Create a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 system source machine.
+. Perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on a minimal {RHEL} 8 instance on the target machine.
+. Restore the backup on the {RHEL} 8 instance.
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you have created a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 source machine, you can install {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 target machine.
+
+* To install {ProjectServer}, see {AdministeringDocURL}: Administering_Project/{Accessing-Server}.
+* To install {SmartProxyServer}, see {AdministeringDocURL}: Administering_Project/{installing-capsule-server}.
+
+
+== Restoring a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you perform a fresh installation of {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 instance of the target machine, you can restore the backup you previously created.
+
+*Procedure*
+
+* Restore an offline or incremental backup on the target machine:
+** To restore an offline backup, see {AdministeringDocURL}: Administering_Project/{Restoring-from-a-Full-Backup}.
+** To restore an incremental backup, see {AdministeringDocURL}: Administering_Project/{Restoring-from-Incremental-Backups}.
+* Update your network configuration to match the target server’s IP address and new host name.
```

Comment:
This needs to happen *before* the restore happens. Not afterwards. (restore will fail if that happens afterwards)